.github/workflows/ci-cd-workflow.yml

```diff
@@ -190,3 +226,40 @@ jobs:
           service: trainee-sync
           cluster: trainee-preprod
           wait-for-service-stability: true
+
+    deploy-prod:
+      name: Deploy
+      environment: prod
+      needs: deploy-pre-prod
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+
+        - name: Configure AWS credentials
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: eu-west-2
+
+        - name: Log in to Amazon ECR
+          id: login-ecr
+          uses: aws-actions/amazon-ecr-login@v1
+
+        - name: Fill in the new image ID in the Amazon ECS task definition
+          id: task-def
+          uses: aws-actions/amazon-ecs-render-task-definition@v1
+          with:
+            task-definition: .aws/task-definition.json
```

Comment:
missing the `-prod`. Line 218 needs `-pre-prod` too.

Reply:
done